swap/swap.go

```diff
@@ -463,6 +463,14 @@ func (s *Swap) processAndVerifyCheque(cheque *Cheque, p *Peer) (*int256.Uint256,
 		return nil, err
 	}
 
+	bounced, err := p.getBouncedCheque()
```

Comment:
`hasBouncedCheque` is probably a more accurate variable name here, what do you think?

Reply:
the `bouncedCheque` variable is not a cheque, therefore it should be called something boolean e.g. `hasBouncedCheque`